mass2/core/misc.py

```diff
@@ -21,20 +22,30 @@ def show(fig: plt.Figure | None = None) -> mo.Html:
     return mo.mpl.interactive(fig)
 
 
-def pickle_object(obj: Any, filename: str) -> None:
+def pickle_object(obj: Any, filename: str | Path) -> None:
     """Pickle the given object to the given filename using dill.
     Mass2 Recipe objects are compatible with `dill` but _not_ with the standard `pickle` module."""
     with open(filename, "wb") as file:
         dill.dump(obj, file)
 
 
-def unpickle_object(filename: str) -> Any:
+def unpickle_object(filename: str | Path) -> Any:
     """Unpickle an object from the given filename using dill."""
     with open(filename, "rb") as file:
         obj = dill.load(file)
         return obj
 
 
+def pickle_stream(obj: Any) -> bytes:
+    """Pickle an object to the given stream using dill."""
```

Comment:
There is no given stream. I don't know what a stream is. I don't understand what this function does. Also It's a one line function, so if I can't understand it from the name, it may be better to not exist so I can just try to read the docs for dill.dumps

Reply:
Per the `dill.dumps` docs, it pickles to a string not a bytes. 
```python
Signature:
dill.dumps(
    obj,
    protocol=None,
    byref=None,
    fmode=None,
    recurse=None,
    **kwds,
)
Docstring:
Pickle an object to a string.
```